### PR TITLE
New boilerplate& seeding script completed 

### PR DESCRIPTION
**Pull request by zz00024**
I want to merge this version of my code into my master branch and please let me know if you spot any problem.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+function component() {
+  const element = document.createElement('div');
+
+  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+
+  return element;
+}
+
+document.body.appendChild(component());
```
Comment (sygessese) [inline src/index.js]: Is there a reason you chose this method? If not, I suggest creating a div within your index.html file and adding an id.

File: server/index.js [hunk 2]
```diff
@@ -1,17 +1,16 @@
-// const express = require('express');
-// const db = require('./db');
-// const app = express();
-// const PORT = 8080;
+const express = require('express');
+const db = require('../database/insertDatabase');
+const app = express();
+const PORT = 8080;
```
Comment (sygessese) [inline server/index.js]: Here is where you define your port number, please change it to a number !8080 or !3000

**Comment by zz00024** [comment]
I have switched this version of code. The current version is this

sygessese <notifications@github.com>於 2019年10月12日 週六，上午9:18寫道：

File: server/index.js [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
+app.use(express.static('public'));
+app.use(express.json());
 
-// //write a get rewuest for it
-// app.get("/", (req, res)=> {
-
-// });
\ No newline at end of file
+app.get( '/server', (req, res)=> {
+  res.send('got the request');
```
Comment (sygessese) [inline server/index.js]: I'm not sure what this is doing, do you plan to access the database with this route? Perhaps a more descriptive url from the client's perspective /item or /images.

**Review by sygessese — CHANGES_REQUESTED** [review]
Few minor tweaks, looks straightforward for now!

**Comment by zz00024** [comment]
I am still working on that part. I have not correct the url yet. The
index.js file in server folder is not done yet.

sygessese <notifications@github.com>於 2019年10月12日 週六，上午9:26寫道：

**Review by sygessese — APPROVED** [review]

**Pull request closed without merging**